geode/src/test/java/org/apache/calcite/adapter/geode/rel/GeodeZipsTest.java

```diff
@@ -160,7 +176,10 @@ public void testSelectLocItem() {
         .explainContains("PLAN=GeodeToEnumerableConverter\n"
             + "  GeodeProject(lat=[ITEM($2, 0)], lon=[ITEM($2, 1)])\n"
             + "    GeodeSort(fetch=[1])\n"
-            + "      GeodeTableScan(table=[[geode, zips]])\n");
+            + "      GeodeTableScan(table=[[geode, zips]])\n")
+        .queryContains(
+            GeodeAssertions.query("SELECT loc[0] AS lat, "
```

Comment:
Do you really need to be that strict and enforce OQL comparison ? 
If there are changes in future all tests would have to be changed

Reply:
What is the harm in adding GeodeAssertions query, adding these did help in getting some edge cases. In case there is a future change these can be useful to ensure that the behavior is expected